client/src/lib/address.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  arbitrum,
+  arbitrumSepolia,
+  base,
+  baseSepolia,
+  mainnet,
+  optimism,
+  optimismSepolia,
+  scroll,
+  scrollSepolia,
+  sepolia
+} from "viem/chains";
+
+const axiomV2QueryAddresses = {
```

Comment:
add comment about what this is mapping to/from

Reply:
added